src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Connection.cs

```diff
@@ -316,7 +324,16 @@ void IConnectionControl.Pause()
         void IConnectionControl.Resume()
         {
             Log.ConnectionResume(ConnectionId);
-            _socket.ReadStart(_allocCallback, _readCallback, this);
+            try
+            {
+                _socket.ReadStart(_allocCallback, _readCallback, this);
```

Comment:
I would rather not add extra code just to avoid an exception.  It would be like checking `File.Exists()` before you call `File.Open()`, which usually isn't worth it.